sdk/src/protocols/cowswap/index.ts

```diff
@@ -1,17 +1,42 @@
 import { allow } from "zodiac-roles-sdk/kit"
 import { allowErc20Approve, oneOf } from "../../conditions"
 import { c, Permission } from "zodiac-roles-sdk"
-import { contracts } from "../../../eth-sdk/config"
-import { BigNumberish } from "ethers"
+import { contractAddressOverrides, contracts } from "../../../eth-sdk/config"
+import { Chain } from "../../types"
 
 const GPv2VaultRelayer = "0xC92E8bdf79f0507f65a392b0ab4667716BFE0110"
 const E_ADDRESS = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE"
 
-const swap = async (options: {
-  sell: (`0x${string}` | "ETH")[]
-  buy?: (`0x${string}` | "ETH")[]
-  feeAmountBp?: number
-}) => {
+const getWethAddress = (chain: Chain) => {
```

Comment:
please add return type

Reply:
It should probably live somewhere else too. Not sure exactly where.

Maybe under something like `utils/getWethAddressByChain`

Reply:
Change signature to `getWethAddressByChain`